### DISCUSSION TITLE
Co robimy z "Shift+klik"?

### DISCUSSION THREAD

**Discussion by azabost** [OpenTTD/team#503] [pl_PL]
@OpenTTD/pl_pl 

Hejka 👋

Zauważyłem, że wiele stringów w kategorii outdated zawierających "Shift+Click" w bazowym języku zostało zmienionych w taki sposób:

>Previous base language text:{BLACK}Buy and refit the highlighted road vehicle. Shift+Click shows estimated cost without purchase
>Base lang string:{BLACK}Buy and refit the highlighted road vehicle. Also press Shift to show cost estimate only

Innymi słowy, wystąpienia "Shift+Click" zostały zmienione na "Also press Shift".

Wydaje mi się, że dotyczy to tylko fragmentów wyjaśniających funkcję klawisza Shift, ale niewykluczone, że jakieś objaśnienia do  klawisza Ctrl również zostały zmienione.

Czy zostawiamy po polsku dotychczasowe tłumaczenia, czy zmieniamy je w analogiczny sposób, np. "Przytrzymaj Shift, aby ..." lub "Przytrzymaj też Shift, aby..." itd.?

Pytam tutaj, bo skoro tych zmian jest tak wiele (strzelam, że ze 100), to pewnie dobrze by było zachować ich spójność, jeśli zdecydujemy się je zmienić.

**Comment by pAter-exe** [comment]
Właśnie zostawiłem sobie to na koniec, bo jest to zmiana bardziej kosmetyczna, a tych fraz jest 60 i zapewne w każdej będzie identyczne tłumaczenie. W [jednej z tych fraz](https://translator.openttd.org/string/openttd-master/pl_PL/STR_OBJECT_BUILD_TOOLTIP) napisałem tak:

> [...] Dodatkowo wciśnięcie Shift wyświetli jedynie szacunkowy koszt

Po zastanowieniu się proponuję:

> [...] Dodatkowe wciśnięcie Shift jedynie wyświetla szacunkowy koszt

Odnośnie Ctrl:

> niewykluczone, że jakieś objaśnienia do klawisza Ctrl również zostały zmienione.

Zmiany z Ctrl były głównie na takiej zasadzie:

>[...] Ctrl+click opens a new viewport on industry location
>[...] Ctrl+Click to open a new viewport on industry location

Po prostu przepuściłem dotychczasowe tłumaczenia. Zastanawiam się jednak jakiego czasu używać - przyszłego, czy teraźniejszego. W całym polskim tłumaczeniu te czasy są pisanie zamiennie i wypadałoby zająć się kiedyś konsekwencją.

Inna kwestia, to czy "klik" powinien być pisany wielką literą - pozostawiłem małe, ale chętnie wysłucham argumentów przeciw ;)

**Reply by azabost** [reply]
Co do czasu teraźniejszego lub przyszłego, to sam nie wiem, co będzie brzmiało bardziej naturalnie. Po angielsku zmienili z teraźniejszego na bezokolicznik ("shows" -> "to show") oraz zdają się używać innego trybu (jakby rozkazujący? nie znam się, sorki) w tych zdaniach, dlatego zamiast:

>"Dodatkowe wciśnięcie Shift jedynie wyświetla szacunkowy koszt"

można jeszcze rozważyć:

> "Dodatkowo **wciśnij** Shift, aby jedynie **wyświetlić** szacunkowy koszt"

Powyższe wydaje mi się najwierniejszym tłumaczeniem bazowego: "Also **press** Shift **to show** cost estimate only" (co nie znaczy, że uważam za konieczność tłumaczenie wszystkiego słowo w słowo). Jak ci się to podoba?

Zastanawiałem się też nad "przytrzymaj" zamiast "wciśnij", ale jest mi z grubsza wszystko jedno.

---

Odnośnie "klik" wielką czy małą, to nie będę się przy niczym upierał. Podejrzewam, że po angielsku pisali wielką po to, aby zwiększyć czytelność i według mnie dzieje się to na kilka sposobów:
- Rzeczy pisane wielką literą bardziej przykuwają uwagę i się lepiej wyróżniają w zdaniu.
- Przez analogię do sposobu opisywania klawiszy (np. w zdaniu "wciśnij klawisz M" napisałbym wielkie M, trochę jak nazwę własną), czyli "klik" jest w zasadzie zastępczą nazwą na klawisz.
    - Na marginesie: czy w OpenTTD można zmienić klawiszologię, tak że np. używa się PPM zamiast LPM i na odwrót? Pytam, bo aż się zastanawiam, dlaczego po angielsku nie piszą "Shift+LMB" i wtedy po polsku byśmy mieli "Shift+LPM". Ale nie wiem, czy to na pewno dla przeciętnego człowieka jest bardziej zrozumiałe. Gdyby zmiana klawiszy LPM i PPM była możliwa, to rozumiałbym intencję stojącą za używaniem określenia "klik", a mianowicie miałaby ona na celu uniezależnienie się od zastosowanego sposobu sterowania.

W związku z powyższym wydaje mi się, że "Klik" byłby trochę bardziej czytelny, ale nie przeszkadza mi obecny zapis. Chętnie poznam przemyślenia twoje lub innych osób.

**Reply by pAter-exe** [reply]
Cały ten "Click" wprowadza pewien problem do naszego tłumaczenia. Po angielsku jest krótko, prosto i wszyscy wiedzą o co chodzi.
Po polsku możemy użyć zarówno czasownika, jak i rzeczownika, co determinuje nam dalszą część zdania, np.:
- "Ctrl+kliknij, aby otworzyć..." (za długo, nie wygląda za dobrze, ale jest tryb rozkazujący jak w pierwszym zdaniu frazy)
- "Ctrl+kliknięcie otwiera..." (krócej)
- "Ctrl+Klik otwiera..." (jeszcze krócej; "Klik" wielką literą jako nazwa przycisku)
- "Ctrl otwiera..." (kiedyś było tak i wszyscy rozumieli, a teraz trzeba pamiętać, żeby kliknąć ;)

Ja jestem za tym, aby te tłumaczenia były możliwie najprostsze - niekoniecznie musimy uczyć gracza korzystania z klawiatury i myszki na zasadzie:
> Wciśnij Ctrl i kliknij, aby otworzyć [...]

O wiele lepiej jest rzucać krótkie hasła:
> Ctrl+Klik otwiera [...] Przytrzymaj Shift, aby oszacować koszt

Ale tutaj mieszamy formy, bo najpierw coś "otwiera", a potem już mówimy graczowi, żeby przytrzymał. A tak właściwie obu klawiszy używa się w ten sam sposób - wciska przed kliknięciem i przytrzymuje.

No i w oryginale z tym Shiftem jest nieco precyzyjniej - **wyświetla** szacunkowy koszt.

A może tak:
> Użyj Ctrl, aby otworzyć [...] Jednocześnie użyj Shift, aby tylko wyświetlić szacunkowy koszt

> Build waypoint on railway. Ctrl+Click to select another waypoint to join. Also press Shift to show cost estimate only
> Zbuduj posterunek na trasie kolejowej. Użyj Ctrl, aby połączyć z innym posterunkiem. Jednocześnie użyj Shift, aby tylko wyświetlić szacunkowy koszt
